pkg/collect/host_xfs_info_linux.go

```diff
@@ -0,0 +1,113 @@
+package collect
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+	"unsafe"
```

Comment:
I'm really not a fan of the use of unsafe here
Can we just try execing `xfs_info` or something?